### PR TITLE
[FIX] account_edi_ubl_cii/bis3: remove deprecated EAS code for Sweden

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
remove deprecated EAS code for Sweden. The code is no longer part of the valid list of codes.

See https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/ and https://docs.peppol.eu/edelivery/codelists/

Also, in the endpoint the Swedish organization number must be used, which is taken from the VAT by removing the country code and the last two digits. You can check it out here: https://organisationsnummer.dev/

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account_edi_ubl_bis3/models/account_edi_format.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
     'TR': 9952,
     'VA': 9953,
 
-    'SE': 9955,
+    'SE': '0007',
```
Comment (JulienVR) [inline addons/account_edi_ubl_bis3/models/account_edi_format.py:45]: I don't think this module is still used but it doesn't hurt I guess

**Review by JulienVR — APPROVED** [review]
I'll ask @smetl or @h4818 to r+ :) Thank you !

**Comment by h4818** [comment]
@JordiMForgeFlow could you please rebase?

**Comment by JordiMForgeFlow** [comment]
@JulienVR @h4818 I have rebased and also added a fix to use correctly the swedish organization number in the endpoint, which is not the same format as the VAT.

I have reached out for support to Peppol:
image

File: addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py [hunk 2]
```diff
@@ -138,6 +138,8 @@ def _get_partner_party_vals(self, partner, role):
             })
         if partner.country_id.code == "LU" and 'l10n_lu_peppol_identifier' in partner._fields and partner.l10n_lu_peppol_identifier:
             vals['endpoint_id'] = partner.l10n_lu_peppol_identifier
+        if partner.country_id.code == "SE" and partner.vat:
+            vals['endpoint_id'] = partner.vat.replace("SE", "")[:-2]
```
Comment (JulienVR) [inline addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py:142]: Don't forget to remove this in 16.2, as the `peppol_endpoint` field exists (and can contain a value different from the vat).

**Review by JulienVR — APPROVED** [review]

**Comment by smetl** [comment]
@robodoo r+

**Comment by smetl** [comment]
@robodoo rebase-ff

**Pull request closed without merging**